source/balances/truffle-config.js

```diff
@@ -56,12 +56,13 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: '0.5.12',
+      version: '0.5.16',
       settings: {
         optimizer: {
           enabled: true,
           runs: 20000,
         },
+        evmVersion: 'petersburg',
```

Comment:
`address(this).balance`
Doesn't compile in a way thats compatible with truffle & ganache on the newest solidity compiler.

Reply:
So it's just because ganache doesnt support the newest EVM opcodes so this just uses an older opcode. I don't think it makes anything inconsistent after compilation, its just switching out 1 new opcode.

Reply:
The issue I read was here...
https://ethereum.stackexchange.com/questions/78597/solidity-0-6-0-addressthis-balance-throws-error-invalid-opcode